### PR TITLE
makers: add vue ft

### PR DESCRIPTION
**Pull request by nemtsov**
Add makers for https://vuejs.org/ (`.vue` filetype).

### CODE REVIEW

File: autoload/neomake/makers/ft/vue.vim [hunk 1]
```diff
@@ -0,0 +1,21 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#vue#EnabledMakers()
```
Comment (mhinz) [inline autoload/neomake/makers/ft/vue.vim]: To make the linters happy, please add `abort` at the end of each `:function` line.

File: autoload/neomake/makers/ft/vue.vim [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+function! neomake#makers#ft#vue#eslint()
+    return {
+        \ 'args': ['--format', 'compact', '--plugin', 'html'],
+        \ 'errorformat': '%E%f: line %l\, col %c\, Error - %m,' .
+        \ '%W%f: line %l\, col %c\, Warning - %m'
+        \ }
+endfunction
+
+function! neomake#makers#ft#vue#eslint_d()
+    return {
```
Comment (mhinz) [inline autoload/neomake/makers/ft/vue.vim]:
```vim
return neomake#makers#ft#vue#eslint()
```

**Comment by nemtsov** [comment]
@mhinz all done. How does it look to you now?

**Comment by mhinz** [comment]
LGTM. I'm just waiting for the tests to finish, although I see no reason why they should fail. :)

**Comment by mhinz** [comment]
Thanks!

**Pull request merged**